### ISSUE TITLE
Setting indent to 0 sets it to the default (24)

### ISSUE THREAD

**Issue by lightsinthesky** [brimdata/react-arborist#51]
due to :

https://github.com/brimdata/react-arborist/blob/ffe240f8df4accb30ce9e3819645a21b0e8af302/packages/react-arborist/src/interfaces/tree-api.ts#L79

**Comment by jameskerr** [comment]
Yeah I suppose that should be allowed. I'm curious. Why don't you want an indent?

**Comment by lightsinthesky** [comment]
Trees where icons and style are used to denote hierarchy vs. indentation. An accordion menu is a basic example.

**Comment by jameskerr** [comment]
Makes sense. I just pushed an update to the v2.0.0-rc.2 branch. Here's a zero indent tree: 
<img width="301" alt="CleanShot 2022-10-24 at 09 22 32@2x" src="https://user-images.githubusercontent.com/3460638/197576338-2de5639a-a8e7-480c-a0cc-a62d67ebae57.png">
